tests/api/controllers/motor/motor.c

```diff
@@ -75,13 +75,13 @@ int main(int argc, char **argv) {
   wb_motor_enable_torque_feedback(motor, TIME_STEP);
   wb_motor_set_control_pid(motor, CONTROL_P, CONTROL_I, CONTROL_D);
   wb_motor_set_available_torque(motor, TORQUE);
-  wb_motor_set_position(motor, M_PI_2);
+  wb_motor_set_position(motor, 100);
 
   wb_robot_step(TIME_STEP);
 
   const double target_position = wb_motor_get_target_position(motor);
-  ts_assert_double_equal(target_position, M_PI_2, "The target position value measured by the motor should be %g and not %g",
-                         M_PI_2, target_position);
+  ts_assert_double_equal(target_position, max_position,
+                         "The target position value measured by the motor should be %g and not %g", M_PI_2, target_position);
```

Comment:
Something looks wrong in this assert, shouldn't it be:
```suggestion
  ts_assert_double_equal(target_position, max_position,
                         "The target position value measured by the motor should be %g and not %g", max_position, target_position);
```